hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/bucket/SetEncryptionKey.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ *  with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.apache.hadoop.ozone.shell.bucket;
+
+import org.apache.hadoop.ozone.client.OzoneBucket;
+import org.apache.hadoop.ozone.client.OzoneClient;
+import org.apache.hadoop.ozone.client.OzoneClientException;
+import org.apache.hadoop.ozone.shell.OzoneAddress;
+import picocli.CommandLine;
+
+import java.io.IOException;
+
+/**
+ * set encryption key of the bucket.
+ */
+@Deprecated
+@CommandLine.Command(name = "set-encryption-key",
+    description = "Set encryption key on bucket")
+public class SetEncryptionKey extends BucketHandler {
```

Comment:
I think it would be better to add this new command somewhere under `ozone debug`, to reduce exposure to users.

Reply:
Currently `ozone debug` is a read-only tool. HDDS-8100 has been on my backlog for a while now to add an `ozone repair` CLI that we can place these types of commands under. If required for this change I can begin working on it.

Reply:
Thanks @errose28.  I think this comment from HDDS-8100:

> We should make sure the node being operated on is in the stopped state

does not apply to this "set bucket encryption" key command.  I guess we could still put it under `repair`, but must distinguish from offline tools in the future.

Reply:
That is true. The current proposal to enforce nodes being stopped while repaired would be to check their PID file, which happens in the shell script before invoking any Java code. Therefore making exceptions to that requirement may be hacky to implement, and the differing requirements for different commands may be confusing. Maybe making this subcommand hidden and leaving it under the bucket subcommand would be best.

Reply:
https://picocli.info/#_hidden_subcommands

Reply:
Thank you @adoroszlai and @errose28 for the review. 

I have retained the command under the `ozone sh bucket` set of commands but enabled the picocli `hidden` argument to `true` for the subcommand `set-encryption-key`.